warehouse/templates/packaging/detail.html

```diff
@@ -91,7 +91,12 @@ <h1 class="package-header__name">
       {% else %}
       <a class="status-badge status-badge--good" href="{{ request.route_path('packaging.project', name=release.project.name) }}">Latest Version</a>
       {% endif %}
-      <p class="package-header__date">{{ humanize(release.created) }}</p>
+      <p class="package-header__date">
+        <span class="status-badge">
+          {{ humanize(release.created) }}
```

Comment:
I think prefixing this with a `Released: ` would be appropriate (not sure I totally follow the discussion in this PR about this).

Reply:
Maybe @yeraydiazdiaz meant prefix instead of suffix and the presence of that would cause the human readable date to be lowercase?

Reply:
I don't see how adding a prefix would make the date lowercase? To be clear, I'm talking about changing this line to something like:

```
Released on: {{ humanize(release.created) }}
```

Reply:
I get that but doesn't `Released on: Just now` or `Released on: About 2 hours ago` sound kinda weird? I think the discussion earlier was talking about passing the optional prefix and if it was a truthy value then we'd make the return value of `convertToReadableText` lowercase. In that case we'd need to pass the prefix only when we're converting the value for the time element in `.package-header__date`.  Maybe just doing `Released:` is the way to go.

Reply:
Yeah, I'll leave it to you to decide what sounds best, but I think the simplest approach possible here is probably ideal. `Last released:` might also work.

Reply:
That sounds like a winner to me